### PR TITLE
Api updates for Sessions (CS2)

### PR DESCRIPTION
**Pull request by Armando-Ibarra**

### CODE REVIEW

File: src/main/java/com/checkout/sessions/source/NetworkTokenSource.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class NetworkTokenSource extends SessionSource {
```
Comment (rsaestrela) [inline src/main/java/com/checkout/sessions/source/NetworkTokenSource.java:16]: Nice stuff!

**Review by rsaestrela — CHANGES_REQUESTED** [review]
Seems to be missing `challenge_cancel_reason_code` from this [spec](nas_spec/components/schemas/Sessions/Acs.yaml). Nice work!

File: src/main/java/com/checkout/sessions/AuthenticationType.java [hunk 2]
```diff
@@ -5,6 +5,8 @@
 public enum AuthenticationType {
 
     @SerializedName("regular")
-    REGULAR
+    REGULAR,
+    @SerializedName("Recurring")
```
Comment (rsaestrela) [inline src/main/java/com/checkout/sessions/AuthenticationType.java:9]: `recurring` is lowercase?

Comment (Armando-Ibarra) [reply src/main/java/com/checkout/sessions/AuthenticationType.java]: yes

**Review by rsaestrela — APPROVED** [review]

**Pull request merged**